### DISCUSSION TITLE
Setting Rate Limiting Policy Using Configuration in YARP

### DISCUSSION THREAD

**Discussion by ProximaB** [dotnet/aspnetcore#59500] [Q&A] [answered]
Is it possible with YARP to set a rate limiting policy using configuration without defining a specific .NET service? I have a requirement to fully configure the rate limiting policy through configuration.

I cannot find information if the YARP support that itself, or it needs to be done on my own.

Here is an example of what I'm trying to achieve:
```json
{
  "RateLimiterPolicy": {
    "FixedWindow": {
      "PermitLimit": 100,
      "QueueLimit": 5
    }
  }
}
```
Thanks in advance for any suggestions.

**Comment by davidfowl** [comment, marked as answer]
This is a great question for https://github.com/microsoft/reverse-proxy/

**Reply by ProximaB** [reply]
Sorry, somehow I ended up on aspnetcore.
